package.json

```
@@ -124,6 +124,10 @@
               ],
               "default": "off",
               "description": "Traces the communication between VS Code and the language server."
+            },
+            "required_version": {
```

```suggestion
            "requiredVersion": {
```
Just to maintain the consistency with other config options here